scripts/concurrent_task_utils.py

```diff
@@ -42,13 +44,52 @@ def log(message, show_time=False):
             python_utils.PRINT(message)
 
 
+class TaskResult(python_utils.OBJECT):
+    """Task result for concurrent_task_utils."""
+
+    def __init__(self, name, failed, trimmed_messages, full_messages):
+        """Constructs a TaskResult object.
+
+        Args:
+            name: str. The name of the task.
+            failed: bool. The boolean value representing whether the task
+                failed.
+            trimmed_messages: list(str). List of error messages that are
+                trimmed to keep main part of messages.
+            full_messages: str|list(str). List of full messages returned by the
+                objects.
+        """
+        self.name = name
+        self.failed = failed
+        self.messages = trimmed_messages
+        self.full_messages = full_messages
+
+    @property
+    def all_messages(self):
```

Comment:
change this to get_report (or something around that) [And it should not be a property!]
also, don't change the fell_message property here!

Reply:
Done.